commit a4faa6e5b8f638a59affcf25b2bdf24b5e8755a0
Author: Johan Hovold <johan@kernel.org>
Date:   Thu Jan 26 12:37:32 2017 +0100

    staging: greybus: loopback: fix broken udelay
    
    commit 33b8807a6fe10d0e675e0704444373a6fad93188 upstream.
    
    The loopback driver allows the user to set a minimum delay of up to one
    second to be inserted between test iterations (i.e. request
    submissions). The delay is currently specified in microseconds and is
    implemented using udelay.
    
    Busy looping for long periods is not just anti-social; udelay must not
    be used for delays longer than a few milliseconds due to the risk of
    integer overflow.
    
    Replace the broken udelay with a usleep_range with a 100 us range for
    short delays (< 20 ms) and otherwise revert to using msleep.
    
    Fixes: b36f04fa9417 ("greybus: loopback: Convert thread delay to microseconds")
    Signed-off-by: Johan Hovold <johan@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1241_a4faa6e5b8f638a59affcf25b2bdf24b5e8755a0.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1241_a4faa6e5b8f638a59affcf25b2bdf24b5e8755a0.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
